package.json

```diff
@@ -4,8 +4,9 @@
   "description": "Automatically validate API requests and responses with OpenAPI 3 and Express.",
   "main": "dist/index.js",
   "scripts": {
-    "compile": "rm -rf dist/ && tsc",
-    "compile:windows": "rmdir dist /s /q & tsc",
+    "prepare": "npm run compile",
+    "compile": "rimraf dist/ && tsc",
+    "compile:windows": "rimraf dist/ && tsc",
```

Comment:
`compile` and `compile:windows` commands are the same, here. Just wanted to confirm that's right (and that the `compile:windows` is still needed with rimraf).

Reply:
`compile:windows` isn't necessary and I did consider removing it, but opted to leave it here for parity with upstream. It doesn't harm anything to keep it around.